app/views/shared/_navigation.html.erb

```diff
@@ -1,11 +1,18 @@
 <nav class="navbar navbar-expand-lg navbar-light bg-light">
-  <%= link_to("Home", home_index_path, class: "navbar-brand") %>
+  <%= render 'home/main_items'  %>
```

Comment:
I think this may need to be moved to right above the `challenges` link line so that it renders correctly. The final html should look something like:
```
<div class="collapse navbar-collapse" id="navbarNavAltMarkup">
    <ul class="navbar-nav">
        <li class="nav-item">
            <a href="/home/index" class="nav-link">Home</a>
        </li>
        <li class="nav-item">
            <a href="/challenges" class="nav-link">Challenges</a>
        </li>
    </ul>
</div>
```